### PR TITLE
Milestone 1

### PR DESCRIPTION
**Pull request by John-Arboleda**
# Project 2 Milestone 1

This pull request is a clone of [TheNextWeb](https://perma.cc/M5ZV-Q2D6) home page as an exercise for **Grid**, **Resposive Desing** and **background linear gradient.**

- Mobile-first design was implemented. Media Queries modify the layout for Tablet and Desktop view.
- Navbar, Highlights section, lastest funding rounds, and footer were created with flexbox.
- Background linear gradient was applied for the highlights section images.

Best,
**John & Emmy**

### CODE REVIEW

**Review by Forison — CHANGES_REQUESTED** [review]
# Status::Changes Required🛑

## 👋 Team,

#### You have done a good on this on so far, however there are few issues we have to address to get this project approved🤗.

* [x]  Pull Request should made up of a title and and short description of what went down on the branch.

* [ ] Let Complete the cloning of the page by adding all section as it appears on the original page. In this project, you are to replicate the archived version of the website [The Next Web](https://perma.cc/M5ZV-Q2D6).

#### All screen sizes

* [x] The logo has a link (`a` tag). the logo should be wrapped in `<a>` tags to make, conventionally logo in navigation would redirect you to the home page. for let redirect to "#"

* [x] Use links (`a` tag) for all the menu options, the hamburger button and the search button.
Let us wrap all menu options, button and hamburger into an `<a>` tags.

* [x] The header sticks to the top of the screen when the user scrolls down. The header should stick to the top of the page thus, assume a fixed position to remain fixed when scrolling.

#### Tablet
* [x] Menu and submenu appear on the header, and the hamburger button disappears. Currently the search button also disappear but it shouldn't. it should be at the far right, but no hamburger.

#### Latest Funding section

* [x] The text excerpt at the bottom, that also has a link (`a` tag). Let us wrap it in an anchor tag.
* [x] The text(excerpt) moves out of the pink section at mobile screen. Put some padding to the bottom to prevent this bug.
> latest funding tiles element are each wrapped in `<a>` tags.

#### Footer
* [x] It contains social media icons centered in a `nav` tag, each of them with a link (`a` tag).
Let us wrap it in an anchor tag.

* [x] Below the social media menu, there is another menu also centered, with a `nav` tag and each option with a link (`a` tag).
Let us wrap footer social media menu and link into anchor tags

#### That is it kindly make the change and submit a re-review💪

#### Happy coding👨‍💻

Addo Forison

File: style.css [hunk 1]
```diff
@@ -0,0 +1,619 @@
...
+  display: none;
+  border: 1px solid black;
+  background-color: black;
+  justify-content: space-between;
+  width: 100%;
+}
+
+.right-black-menu,
+.left-black-menu {
+  display: flex;
```
Comment (Forison) [inline style.css:60]: **[OPTIONAL]**
* [ ] Avoid redefining a same attribute over and over, you define it, in isolation and used wherever you want to. We could declare this as a class on it own. 
`.flex{
display: flex;
}
`

**Review by Forison — COMMENTED** [review]
# Status::Changes Required🛑

## 👋 Team,

#### You have done a good on this on so far, however there are few issues we have to address to get this project approved🤗.

* [x]  Pull Request should made up of a title and and short description of what went down on the branch.

* [ ] Let Complete the cloning of the page by adding all section as it appears on the original page. In this project, the student replicates the archived version of the website [The Next Web](https://perma.cc/M5ZV-Q2D6).

#### All screen sizes

* [x] The logo has a link (`a` tag). the logo should be wrapped in `<a>` tags to make, conventionally logo in navigation would redirect you to the home page. for let redirect to "#"

* [x] Use links (`a` tag) for all the menu options, the hamburger button and the search button
Let us wrap all menu options, button and hamburger into an `<a>` tags

* [x] The header sticks to the top of the screen when the user scrolls down. The header should stick to the top of the page thus, assume a fixed position to remain fixed when scrolling.

#### Mobile

* [x] There is a hamburger (bars) icon at the rightmost part (next to search icon). T

#### Tablet
* [x] Menu and submenu appear on the header, and the hamburger button disappears Currently the search button also disappear but it shouldn't. it should be at the far right, but no hamburger.

#### Mobile

All the first 3 articles take the maximum width and are are aligned in one column. 

#### Latest Funding section

* [x] The text excerpt at the bottom, that also has a link (`a` tag). Let us wrap it in an anchor tag.
* [x] The text(excerpt) moves out of the pink section at mobile screen. Put some padding to the bottom to prevent this bug.

#### Footer
* [x] It contains social media icons centered in a `nav` tag, each of them with a link (`a` tag)
Let us wrap it in an anchor tag.

* [x] Below the social media menu, there is another menu also centered, with a `nav` tag and each option with a link (`a` tag)
Let us wrap footer link into anchor tags

#### That is it kindly make the change and submit a re-review💪

#### Happy coding👨‍💻

Addo Forison

**Comment by mimipeshy** [comment]
### STATUS :APPROVED :partying_face: 

Hey there  @John-Arboleda and @francisuloko   :wave: :wave:  good job on working on the requested changes. This project is therefore approved. Go ahead and merge. All the best in the next milestones :man_technologist: 

Regards
Peris Ndanu

**Pull request merged**